msgraph/graph_service_client.py

```diff
@@ -60,3 +62,12 @@ def me(self) -> UserItemRequestBuilder:
         url_tpl_parameters["user%2Did"] = "me-token-to-replace"
 
         return UserItemRequestBuilder(self.request_adapter, url_tpl_parameters)
+
+    @property
+    def batch(self) -> BatchRequestBuilder:
```

Comment:
I think there's an import missing. Lint checks are not passing with this change...

Reply:
Yes, this is because core is not released, please check the PR on core to enable release

Reply:
You would still need an import for the `BatchRequestBuilder` type yes?

Reply:
I think Andrew meant in this file as well (in addition to the dependency you've just updated)

Reply:
We use differed type for the annotation, then this is found, now that we have done the release